R/deployApp.R

```diff
@@ -520,37 +520,20 @@ bundleApp <- function(appName, appDir, appFiles, appPrimaryDoc, assetTypeName,
                       isCloudServer = FALSE, metadata = list(), image = NULL) {
   logger <- verboseLogger(verbose)
 
-  quartoInfo <- inferQuartoInfo(
+  logger("Inferring App mode and parameters")
+  metadata <- appMetadata(
```

Comment:
`metadata` is a function argument; you're overwriting the incoming value.

Reply:
I'll call this `appMetadata` to be consistent with the function name.